src/nav/infrastructure.yml

```diff
@@ -52,8 +52,10 @@ pages:
             path: /docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux
           - title: Configuration management tools
             path: /docs/infrastructure/install-infrastructure-agent/linux-installation/install-configuration-management-tools
-          - title: Container instrumentation
```

Comment:
Let's delete this title once we confirm how we want to frame it. I think "Monitor containers" is slightly more accurate than "instrument your host containers" (since the doc does say we don't need to do anything extra

Reply:
agree